src/main/java/org/terasology/launcher/game/TerasologyGameVersions.java

```diff
@@ -93,7 +93,7 @@ public synchronized TerasologyGameVersion getGameVersionForBuildVersion(GameJob
     }
 
     public synchronized void loadGameVersionsFromPackageManager(GameSettings gameSettings, Path launcherDirectory, Path gameDirectory) {
-        final Path cacheDirectory = launcherDirectory.resolve(DirectoryUtils.CACHE_DIR_NAME);
+        final Path cacheDirectory = launcherDirectory.resolve(LauncherDirectoryUtils.CACHE_DIR_NAME);
```

Comment:
I think all the _launcher managed directories_ should be retrieved from the config/settings object, instead of resolving the path again and again.

Reply:
I agree with that.. It should look more sensible to have the `PackageManager` directly depend on the settings object and resolve all required paths itself.